src/group_theory/quotient_group.lean

```diff
@@ -412,4 +413,25 @@ monoid_hom.to_mul_equiv
 
 end third_iso_thm
 
+
+section trivial
+
+/-These first few lemmas should maybe go somewhere else, where?-/
+
```

Comment:
This will need to be removed, once it is resolved.

Reply:
Done